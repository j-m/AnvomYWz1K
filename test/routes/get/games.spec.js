const request = require('supertest')

const app = require('../../../src/app/koa.js')

describe('routes get games', () => {
  test('should contain templated text', async () => {
    const response = await request(app.callback()).get('/games/')
    expect(response.body).toMatchSnapshot()
  })
})
