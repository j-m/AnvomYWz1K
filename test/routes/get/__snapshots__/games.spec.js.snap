// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes get games should contain templated text 1`] = `Object {}`;
